Patch for Squid bug #2203:

LDFLAGS gets unset when --with-filedescriptors=n is added to
SQUID_CONFIGURE_ARGS.

--- configure.orig	2008-03-26 21:16:54.000000000 +0100
+++ configure	2008-03-26 21:18:40.000000000 +0100
@@ -43819,12 +43819,12 @@
 
 { echo "$as_me:$LINENO: checking Maximum number of filedescriptors we can open" >&5
 echo $ECHO_N "checking Maximum number of filedescriptors we can open... $ECHO_C" >&6; }
+TLDFLAGS="$LDFLAGS"
 if test -n "$squid_filedescriptors_num" ; then
   SQUID_MAXFD=$squid_filedescriptors_num
   { echo "$as_me:$LINENO: result: $SQUID_MAXFD (user-forced)" >&5
 echo "${ECHO_T}$SQUID_MAXFD (user-forced)" >&6; }
 else
-  TLDFLAGS="$LDFLAGS"
   case $host in
   i386-unknown-freebsd*)
       if echo "$LDFLAGS" | grep -q pthread; then
